validate/validate.go

```diff
@@ -35,17 +48,17 @@ type Options struct {
 
 // HeaderOptions represents validation options for a TDX attestation Quote Header.
 type HeaderOptions struct {
-	// MinimumQeSvn is the minimum QE security version number. Must be nil or 2 bytes long. Not checked if nil.
-	MinimumQeSvn []byte
-	// MinimumPceSvn is the minimum PCE security version number. Must be nil or 2bytes long. Not checked if nil.
-	MinimumPceSvn []byte
+	// MinimumQeSvn is the minimum QE security version number. Not checked if nil.
```

Comment:
These should be uint16 since Go can express that type.

Reply:
Done.